### ISSUE TITLE
[CS 1.6] About view models.

### ISSUE THREAD

**Issue by zNeks** [ValveSoftware/halflife#2502]
"Textures on hand's bodys can be changed like in CSO with .tga files". Can it be done?

**Comment by CS-PRO1** [comment]
I don't believe it's possible with the old Goldsrc .mdl system which heavily depends on textures that are either compiled with the model in 1 .mdl file, or having external textures (textures only without meshes) .mdl files with _T.mdl extension.

**Comment by SamVanheer** [comment]
Model rendering code exists in both the engine and game code, and this will likely require an interface change to work, so existing games and mods could break.

There is probably a lot of mod code that assumes models work like they do now as well, suddenly having external texture files loaded from somewhere else could result in unexpected behavior.

**Comment by bereke1998** [comment]
me too find about it talk, i think it's possible

**Comment by zNeks** [comment]
I didn't find it.

**Comment by regener** [comment]
I just wanted to request the same.
Changing skin for viewmodels would be really useful.
